src/ParseServer.js

```diff
@@ -191,6 +189,17 @@ class ParseServer {
 
     const dbInitPromise = databaseController.performInitialization();
 
+    if (Object.keys(oauth).length > 0) {
+      /* eslint-disable no-console */
+      console.warn('oauth option is deprecated and will be removed in a future release, please use auth option instead');
```

Comment:
is it possible or desirable to use the logger here?  Maybe you just expect users to see this during dev/testing?  reasonable i guess, but would still see with the logger.  maybe logger isn't available yet?